t/v3.t

```diff
@@ -208,3 +206,68 @@ qr/1:.*"created":true.*
 3:.*"value":"bcd4a".*
 timeout/
 --- timeout: 5
+
+
+=== TEST 5: setx(key, val) failed
```

Comment:
we can use `reindex` tool to format the test cases: https://github.com/openresty/openresty-devel-utils/blob/master/reindex